src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/openapi/TokenCertificatesApiController.java

```diff
@@ -63,6 +63,32 @@ public TokenCertificatesApiController(TokenCertificateService tokenCertificateSe
         this.tokenCertificateConverter = tokenCertificateConverter;
     }
 
+    @Override
+    @PreAuthorize("hasAuthority('ACTIVATE_TOKEN')")
```

Comment:
We do not have `ACTIVATE_TOKEN` permission in permissions.yml (anymore?).

Instead there is `activate_disable_auth_cert` and `activate_disable_sign_cert`.

Controller should still check for "activate_disable_auth_cert or activate_disable_sign_cert", but the more strict, cert type specific check needs to be done in addition to that. 

Permission check needs to know which type is in question. Probably this must to be done in service, see https://github.com/nordic-institute/X-Road/blob/1b453f07225d5b279c4a9a60a82cdf4354fd6102/src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/TokenCertificateService.java#L301-L303 (maybe refactor so that common parts are shared?)

Reply:
Yes, ACTIVATE_TOKEN is there https://github.com/nordic-institute/X-Road/blob/develop/src/proxy-ui-api/src/main/resources/permissions.yml#L54. Should it be removed?

I'll change the controller checks and use that verify.

Reply:
Sorry, yeah, I don't know why I said it was not there! 

It is not the correct permission anyhow, since this API is (de)activates a cert, not a token. So, activate_disable_auth_cert and activate_disable_sign_cert are the correct ones.

I will look into the activate_token permission removal myself: https://niis.slack.com/archives/GGBHVCYSY/p1577696790001400

Reply:
is fixed now